test/Controller/PromotionalCollectionControllerTest.php

```diff
@@ -0,0 +1,527 @@
+
+        $this->assertSame('Promotional collection title | Highlights | eLife', $crawler->filter('title')->text());
+        $this->assertSame('/highlights/1/promotional-collection-title', $crawler->filter('link[rel="canonical"]')->attr('href'));
+        $this->assertSame('http://localhost/highlights/1/promotional-collection-title', $crawler->filter('meta[property="og:url"]')->attr('content'));
+        $this->assertSame('Promotional collection title', $crawler->filter('meta[property="og:title"]')->attr('content'));
+        $this->assertSame('Promotional collection impact statement', $crawler->filter('meta[property="og:description"]')->attr('content'));
+        $this->assertSame('Promotional collection impact statement', $crawler->filter('meta[name="description"]')->attr('content'));
+        $this->assertSame('article', $crawler->filter('meta[property="og:type"]')->attr('content'));
+        $this->assertSame('summary_large_image', $crawler->filter('meta[name="twitter:card"]')->attr('content'));
+        $this->assertSame('https://www.example.com/iiif/ban%2Fner/full/2000,/0/default.jpg', $crawler->filter('meta[property="og:image"]')->attr('content'));
+        $this->assertSame('2000', $crawler->filter('meta[property="og:image:width"]')->attr('content'));
+        $this->assertSame('1333', $crawler->filter('meta[property="og:image:height"]')->attr('content'));
+        $this->assertSame('promotional-collection/1', $crawler->filter('meta[name="dc.identifier"]')->attr('content'));
+        $this->assertSame('elifesciences.org', $crawler->filter('meta[name="dc.relation.ispartof"]')->attr('content'));
+        $this->assertSame('Promotional collection title', $crawler->filter('meta[name="dc.title"]')->attr('content'));
+        $this->assertEmpty($crawler->filter('meta[name="dc.description"]'));
+        $this->assertSame('2010-01-01', $crawler->filter('meta[name="dc.date"]')->attr('content'));
+        $this->assertSame('© 2010 eLife Sciences Publications Limited. This article is distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use and redistribution provided that the original author and source are credited.', $crawler->filter('meta[name="dc.rights"]')->attr('content'));
+    }
+
+    /**
+     * @test
+     * @dataProvider incorrectSlugProvider
+     */
+    public function it_redirects_if_the_slug_is_not_correct(string $slug = null, string $queryString = null)
+    {
+        $client = static::createClient();
+
+        $url = "/highlights/1{$slug}";
+
+        $expectedUrl = $this->getUrl();
+        if ($queryString) {
+            $url .= "?{$queryString}";
+            $expectedUrl .= "?{$queryString}";
+        }
+
+        $client->request('GET', $url);
+
+        $this->assertTrue($client->getResponse()->isRedirect($expectedUrl));
+    }
+
+    /**
+     * @test
+     */
+    public function it_displays_a_404_if_the_promotional_collection_is_not_found()
```

Comment:
there may be another 404 test where:
- given the promotional collection is found
- and the metrics (page views) are not found
- when the promotional collection page is loaded
- then the page is a 200
- and something is displayed in place of `Views: ` (or maybe it just disappears?)

Reply:
I have removed the page views from promotional collection pages